blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/BluefloodServiceStarter.java

```diff
@@ -295,8 +296,10 @@ public void run() {
                     TimeUnit.MINUTES.toMillis(savePeriodMins));
         }
 
-        // has the side-effect of causing static initialization of Metrics, starting instrumentation reporting.
-        new RestartGauge(Metrics.getRegistry(), RollupService.class);
+        if (!SkipInstantiateRestartGauge) {
+            // has the side-effect of causing static initialization of Metrics, starting instrumentation reporting.
+            new RestartGauge(Metrics.getRegistry(), RollupService.class);
```

Comment:
Probably, but I don't know what the necessary logic would be. I don't really understand everything that this class is doing. It's a major smell in my opinion that it's instantiating an object but not storing a reference to it, a "naked `new`". There's a lot of deep magic underneath. At this point I'm just trying to isolate portions of the system to test them.